extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -461,5 +461,10 @@ export const connectionStringSettingPlaceholder = localize('connectionStringSett
 export const noAzureFunctionsInFile = localize('noAzureFunctionsInFile', "No Azure functions in the current active file");
 export const noAzureFunctionsProjectsInWorkspace = localize('noAzureFunctionsProjectsInWorkspace', "No Azure functions projects found in the workspace");
 export const addPackage = localize('addPackage', "Add Package");
+export const createNewLocalAppSetting = localize('createNewLocalAppSetting', '$(add) Create new local app setting');
```

Comment:
What's `$(add)`?  string replacement macro for azure functions?

Reply:
https://code.visualstudio.com/api/references/icons-in-labels#icons-in-labels

@kisantia I would suggest not including the icons in the localized strings though as those should never be localized. And since they're icons they typically should show up in a specific place (before the text) regardless of what the localized text is like so it should be fine to make a separate constant for the text + icon such as I did here : https://github.com/microsoft/azuredatastudio/blob/main/extensions/sql-database-projects/src/common/constants.ts#L118